test/powershell/Modules/Microsoft.PowerShell.Utility/Invoke-Item.Tests.ps1

```diff
@@ -72,10 +72,13 @@ Describe "Invoke-Item basic tests" -Tags "Feature" {
                 Stop-Process -InputObject $notepadProcess
             }
         } else {
-            ## On Unix, we use `UseShellExecute = false`
-            ## 'ping' on Unix write out usage to stderr
-            & $powershell -noprofile -c "Invoke-Item '$ping'" 2> $redirectFile
-            Get-Content $redirectFile -Raw | Should Match "usage: ping"
+            ## We are using ForEach loop as there might be multiple copies of ping.
+            ForEach ($ping_path in $ping) {
```

Comment:
Doesn't seem like there's value in running the test for multiple copies of `ping`.  It appears that all this test does is run an executable and check that the usage information is collected by PowerShell.  So an alternate way to fix this is to change line 54 to:

```powershell
$ping = (Get-Command "ping" -CommandType Application | Select-Object -First 1).Source
```

This will just get the first one if there's more than one.  The unnecessary `ForEach-Object` was only there to return the `Source` property so we can simplify that as well.

Reply:
Done.